substrate/frame/nfts/src/features/create_delete_collection.rs

```diff
@@ -66,6 +66,11 @@ impl<T: Config<I>, I: 'static> Pallet<T, I> {
 
 		CollectionConfigOf::<T, I>::insert(&collection, config);
 		CollectionAccount::<T, I>::insert(&owner, &collection, ());
+
+		if let Some(max_supply) = config.max_supply {
+			Self::deposit_event(Event::CollectionMaxSupplySet { collection, max_supply });
```

Comment:
Is this emitted before or after the Collection Creation event?

Reply:
Before, you think it's better to put it after?

Reply:
Yea, otherwise someone listening for events would possibly not be able to match it up.